ir/src/main/scala/org/scalajs/core/ir/Printers.scala

```diff
@@ -895,6 +888,15 @@ object Printers {
     protected def print(c: Int): Unit =
       out.write(c)
 
+    protected def print(optimizerHints: OptimizerHints)(
+        implicit dummy: DummyImplicit): Unit = {
+      if (optimizerHints != OptimizerHints.empty) {
+        print("@hints(")
+        print(optimizerHints.bits)
```

Comment:
Hum that's weird too, actually. Isn't it calling the `print(c: Int)` just above? There's probably a `.toString()` missing.